worlds/pokemon_rb/docs/setup_fr.md

```diff
@@ -0,0 +1,102 @@
+# Guide de configuration pour Pokémon Rouge et Bleu : Archipelago
+
+## Important
+
+Comme nous utilisons BizHawk, ce guide s'applique uniquement aux systèmes Windows et Linux.
+
+## Logiciel requis
+
+- BizHawk : [Version BizHawk de TASVideos](https://tasvideos.org/BizHawk/ReleaseHistory)
+   - Les versions 2.3.1 et ultérieures sont prises en charge. La version 2.7 est recommandée pour des raisons de stabilité.
```

Comment:
I just opened a pull request to change the recommended Bizhawk version to 2.9 in the existing setup guides